usermembership/tool/src/webapp/usermembership/userlist.jsp

```diff
@@ -148,15 +148,19 @@
 		                <h:outputText value="#{msgs.user_created_on}"/>
 		            </t:commandSortHeader>
 		        </f:facet>
-				<h:outputText value="#{row.createdOn}"/>
+				<h:outputText value="#{row.createdOn}">
+					<f:convertDateTime pattern="#{msgs.date_format}"/>
+				</h:outputText>
 			</h:column>
 			<h:column id="modifiedOn">
 				<f:facet name="header">
 		            <t:commandSortHeader columnName="modifiedOn" immediate="true" arrow="true">
 		                <h:outputText value="#{msgs.user_modified_on}"/>
 		            </t:commandSortHeader>
 		        </f:facet>
-				<h:outputText value="#{row.modifiedOn}"/>
+				<h:outputText value="#{row.modifiedOn}">
+					<f:convertDateTime pattern="#{msgs.date_format}"/>
```

Comment:
I think this also needs to be timezone aware as the User date isn't timezone aware and the previous formatting in the `Time` class looks to compensate for the user's timezone. I haven't tested this though.